src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -37,6 +37,11 @@ import VariantCountCache from "./VariantCountCache";
 
 type PageMode = 'patient' | 'sample';
 
+class OncokbDefault implements IOncoKbData {

```

Comment:
I would make this a function -- lint complains about multiple classes in a single file :)
```
function defaultOncoKbData(): IOncoKbData
```